apps/src/sites/code.org/pages/public/teacher-dashboard/index.js

```diff
@@ -672,11 +670,9 @@ function main() {
 
     $scope.assessmentsLoaded = false;
     $scope.assessmentStages = [];
-    $scope.assessments = sectionsService.assessments({id: $routeParams.id});
 
     $scope.surveysLoaded = false;
     $scope.surveyStages = [];
-    $scope.surveys = sectionsService.surveys({id: $routeParams.id});
```

Comment:
It looks like other nearby lines are also unused:
- `assessmentsLoaded`
- `assessmentStages`
- `surveysLoaded`
- `surveyStages`

Can we tear those out while we're at it?

Reply:
There is a ton of code to delete in this file. Going to follow up with the tabs work I'm doing next week.